server/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,10 +15,13 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('first_name');
```

Comment:
We already fix this before. Please retain the original migration as it is, if you have updates in user table. Make another migration file.